vms/avm/vm.go

```diff
@@ -512,18 +571,13 @@ func (vm *VM) GetUTXOs(
 				}
 				utxos = append(utxos, utxo)
 				seen.Add(utxoID)
-				lastAddr = addr
-				lastIndex = utxoID
-				currLimit--
-				if currLimit <= 0 {
-					break // Found [currLimit] utxos; stop current set.
-				}
 			}
+			addr = lastAddr
```

Comment:
I don't think it makes sense/is needed to set this here... Right? It feels like it could cause this function to call `vm.state.Funds` on the previously called address.

Reply:
Yep that makes sense - no point doing it 👍